core/chains/evm/client/simulated_backend_client.go

```diff
@@ -1,4 +1,4 @@
-package cltest
+package client
```

Comment:
argument for this is: we have lots of components which use client.Client, we should make the geth sim backed implementation of the interface available in the same package so no additional imports are needed. Aside from testing very specific client.Client return errors, its preferable to encourage testing with the real sim vs trying to mock specific responses since its fast and extremely high fidelity.